src/main/java/kobot/task/TaskList.java

```diff
@@ -0,0 +1,90 @@
+package kobot.task;
+
+import kobot.task.Deadline;
+import kobot.task.Event;
+import kobot.task.Task;
+import kobot.task.ToDo;
+
+public class TaskList {
+    private static final int MAX_TASK_COUNT = 100;
+    private Task[] taskList = new Task[MAX_TASK_COUNT];
+    private int taskCount = 0;
+
+    /**
+     * Adds a to-do item to the task list.
+     *
+     * @param description Description of the task.
+     */
+    public void addToDo(String description) {
+        taskList[taskCount] = new ToDo(description);
+        System.out.println("Task has been added to list:");
+        System.out.println(taskList[taskCount]);
+        taskCount++;
+    }
+
+    /**
+     * Adds a task that needs to be done by a specific date/time to the task list.
+     *
+     * @param description Description of the task.
+     * @param by Date/Time that the task has to be completed by.
+     */
+    public void addDeadline(String description, String by) {
+        taskList[taskCount] = new Deadline(description, by);
+        System.out.println("Deadline has been added to list:");
+        System.out.println(taskList[taskCount]);
+        taskCount++;
+    }
+
+    /**
+     * Adds a task that starts and ends at a specific date/time.
+     *
+     * @param description Description of the task.
+     * @param from Date/Time that the task starts.
+     * @param to Date/Time that the task ends.
+     */
+    public void addEvent(String description, String from, String to) {
+        taskList[taskCount] = new Event(description, from, to);
+        System.out.println("Event has been added to list:");
+        System.out.println(taskList[taskCount]);
+        taskCount++;
+    }
```

Comment:
Maybe can extract the last 3 lines out from these 3 methods and put into a new method to make it less repetitive